netlify.toml

```diff
@@ -5,7 +5,33 @@
 [build.environment]
   YARN_FLAGS = "--frozen-lockfile"
 
+# Redirect HTTP to HTTPS
+# TODO: Delete when Codetribute successfully moves to codetribute.mozilla.org
+[[redirects]]
+  from = "http://codetribute.netlify.com/*"
+  to = "https://www.codetribute.netlify.com/:splat"
```

Comment:
Why is there a `www` here?

Reply:
Right, but that only applies to top-level domains, e.g. `mozilla.com`. It would not apply to `codetribute.mozilla.com`. You can remove the `www` from the redirect.

Reply:
Done.